infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/impl/AbstractSszComposite.java

```diff
@@ -95,7 +96,7 @@ protected IntCache<SszChildT> createCache() {
   }
 
   @Override
-  public final SszChildT get(final int index) {
+  public SszChildT get(final int index) {
```

Comment:
Didn't find where it's overridden

Reply:
this is probably due to several back and forth and I haven't rolled it back :)